velox/vector/BaseVector.h

```diff
@@ -654,6 +669,14 @@ class BaseVector {
   }
 
  protected:
+  /// Returns a brief summary of the vector. The default implementation includes
+  /// encoding, type, number of rows and number of nulls.
+  ///
+  /// For example,
+  ///     [FLAT INTEGER: 3 elements, no nulls]
+  ///     [DICTIONARY INTEGER: 5 elements, 1 nulls]
+  virtual std::string toSummaryString() const;
```

Comment:
nit, any reason we cant make this public ?

Reply:
I don't think there is a need. toString() and toString(true) methods seem to be sufficient. Am I missing something?